Patches/Mister Meeseeks/Meeseek_Race.xml

```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+	<Operation Class="PatchOperationFindMod">
+		<mods>
+			<li>Mister Meeseeks</li>
+		</mods>
+		<match Class="PatchOperationSequence">
+			<operations>
+
+				<!--Test for comps-->
+				<li Class="PatchOperationSequence">
+				  <success>Always</success>
+				  <operations>
+				    <li Class="PatchOperationTest">
+					  <xpath>/Defs/AlienRace.ThingDef_AlienRace[defName="MeeseeksRace"]/comps</xpath>
+					  <success>Invert</success>
+					</li>
+					
+					<li Class="PatchOperationAdd">
+					  <xpath>/Defs/AlienRace.ThingDef_AlienRace[defName="MeeseeksRace"]</xpath>
+					  <value><comps /></value>
+					</li>
+					
+				  </operations>
+				</li>
+
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/ThingDef[defName="CM_Meeseeks_Box_Thing_Meeseeks_Box"]/statBases</xpath>
+				<value>
+					<Bulk>5</Bulk>
+				</value>
+			</li>
+
+			<li Class="PatchOperationAddModExtension">
+				<xpath>/Defs/AlienRace.ThingDef_AlienRace[defName="MeeseeksRace"]</xpath>
+				<value>
+					<li Class="CombatExtended.RacePropertiesExtensionCE">
+						<bodyShape>Humanoid</bodyShape>
+					</li>
+				</value>
+			</li>
+
+				<!-- Gizmos -->
+				<li Class="PatchOperationAdd">
+					<xpath>/Defs/AlienRace.ThingDef_AlienRace[defName="MeeseeksRace"]/comps</xpath>
+					<value>
+						<li>
+							<compClass>CombatExtended.CompPawnGizmo</compClass>
+						</li>
+					</value>
+				</li>
+
+
+				<!--Meseeks-->
+
+				<li Class="PatchOperationAdd">
+					<xpath>/Defs/AlienRace.ThingDef_AlienRace[defName="MeeseeksRace"]/statBases</xpath>
+					<value>
+						<MeleeDodgeChance>1</MeleeDodgeChance>
+						<MeleeCritChance>1</MeleeCritChance>
+						<MeleeParryChance>1</MeleeParryChance>
+						<SmokeSensitivity>0</SmokeSensitivity>
```

Comment:
I don't think they should be immune to smoke. They're organic, as far as I understand,